### PR TITLE
Improve visual appearance of external link message #1578

### PR DESCRIPTION
**Pull request by shljessie**
Fixing Issue #1578 Improving visual appearance of external link message.
Box shadow appears on hover

Before
Screen Shot 2022-07-12 at 4 38 51 PM

After
Screen Shot 2022-07-12 at 4 34 46 PM

### CODE REVIEW

**Comment by lizadaly** [comment]
Ah so much better with padding! Doing the shadow on hover is a nice touch too. 

I think we might still want the informational text about how it's an external resource, like in a small heading? Tagging @cath9 for her opinion too.

File: web/frontend/styles/casebook.scss [hunk 1]
```diff
@@ -398,11 +398,35 @@ form.edit_content_resource, form.edit_content_section, form.edit_content_caseboo
     }
   }
 
-  /* TODO: this block is duplicated in TheResource.vue; factor out at some point */
+  .link-container :hover{
```
Comment (rebeccacremona) [inline web/frontend/styles/casebook.scss]: Just a friendly reminder to include styles for keyboard-only users too 🙂

Comment (shljessie) [reply web/frontend/styles/casebook.scss]: Oops sorry 😅 will include that!

**Comment by shljessie** [comment]
Seems like there is a default keyboard focus outline that shows! Im having trouble getting the box-shadow to work for the keyboard focus. There seems to be some difficulty doing this with input boxes too. https://stackoverflow.com/questions/10692421/box-shadow-on-focus 
Wondering if this would be okay? @rebeccacremona 

Screen Shot 2022-07-13 at 10 32 44 AM

**Comment by shljessie** [comment]
Final Designs! 
Added link icon in front and a clickable box description

Screen Shot 2022-07-13 at 2 33 41 PM

Screen Shot 2022-07-13 at 2 33 49 PM

**Comment by shljessie** [comment]
Screen Shot 2022-07-13 at 2 58 47 PM
 Wondering if we should just have the link icon

**Comment by lizadaly** [comment]
I like this last iteration!

**Comment by cath9** [comment]
Agree!

**Comment by rebeccacremona** [comment]
> Im having trouble getting the box-shadow to work for the keyboard focus.

I think that's because the focus lands on the `<a href="....>` inside the container, rather than the container. How do you like
```
  .link-container:hover .link-resource, .link-container a:focus .link-resource {
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  }

  .link-container a:focus {
    outline: none;
  }
```
(same box shadow on focus as on hover)
or 
```
  .link-container {
    &:hover .link-resource {
      box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
    }

    a:focus {
      outline: none;

      .link-resource {
        @include generic-focus-styles;
      }
    }
  }
```
(same focus css as used elsewhere on the page/site)
image

**Comment by rebeccacremona** [comment]
That tweaks the hover too, from (on my Chrome)
image
to 
image

**Comment by rebeccacremona** [comment]
Also idly wondering whether we can lose "click to view" language entirely. Maybe intuitive, with your design?

**Comment by lizadaly** [comment]
I agree that now the design doesn't necessarily need the text though I also think it looks fine with. Your call!

**Review by lizadaly — APPROVED** [review]

**Comment by lizadaly** [comment]
(I can fix the conflict thing when we're ready to merge.)

**Comment by shljessie** [comment]
Yup changes have been pushed added shadow as default and shadow gets darker when hovered/focused. 


Screen Shot 2022-07-13 at 4 26 25 PM

Screen Shot 2022-07-13 at 4 26 34 PM

**Comment by lizadaly** [comment]
Hopefully someday we can use `text-wrap: balance` to get rid of that orphaned 59 just hanging out there on its own line: https://twitter.com/LeaVerou/status/983304306978902016?s=20&t=yxtFcMyEMSIOFQG6hHGR9w

**Comment by rebeccacremona** [comment]
I think that for the darkening to work how you want in Chrome and not have it look like that screenshot I posted,
```
  .link-container {
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  }
```
needs to be
```
  .link-container .link-resource {
    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
  }
```

**Review by rebeccacremona — APPROVED** [review]

**Pull request merged**